tests/rabota_by_tests.py

```diff
@@ -0,0 +1,35 @@
+"""
+Module contains tests for jobs.tut.by that is written with
+parser_jobs_tut_by.py and rabota_fixtures.by
+"""
+
+
+def test_connection(get_response):
+    assert get_response.status_code == 200, 'Connection error'
+
+
+def test_shotgun_not_found(get_query_shotgun):
+    assert 'По запросу «shotgun» ничего не найдено' in get_query_shotgun.text, 'Something going wrong for "shotgun" query'
```

Comment:
'Something going wrong for "shotgun" query' - from this message I don't understand what is going wrong if test fails

Reply:
Fixed